index.html

```diff
@@ -1170,9 +1150,124 @@
             all remaining steps.
             </li>
             <li>Otherwise, the user <em>granted permission</em> to use a
```

Comment:
Not sure why we use past tense here, "user grants permission" should be enough. Same thing in "user denied permission" few lines before.